src/client/lib/PanelComponents.ts

```diff
@@ -59,26 +59,30 @@ export class AuthProfileTreeItem extends vscode.TreeItem {
         }
     }
     private static createTooltip(profile: AuthProfileListing): string {
-        let tooltip = `${profile.Kind}: `;
+        let tooltip = `Profile Kind: ${profile.Kind}: `;
```

Comment:
TS idiomatics nit: I'd probably push the tooltip line fragments into an array ```tooltips = [];```, and at the end, do a join:
```return tooltips.join('\n');```

Reply:
updated